bundle/analyze/superset/superset-init.sh

```diff
@@ -37,52 +34,43 @@ Init Step ${1}/${STEP_CNT} [${2}] -- ${3}
 
 EOF
 }
-ADMIN_PASSWORD="admin"
-# If Cypress run – overwrite the password for admin and export env variables
-if [ "$CYPRESS_CONFIG" == "true" ]; then
-    ADMIN_PASSWORD="general"
-    export SUPERSET_CONFIG=tests.superset_test_config
-    export SUPERSET_TESTENV=true
-    export ENABLE_REACT_CRUD_VIEWS=true
-    export SUPERSET__SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://superset:superset@db:5432/superset
+
+# Install drivers
+if [ ! -z "${SUPERSET_ADDITIONAL_DEPENDENCIES}" ]; then
+  echo_step "1" "Starting" "Install additional dependencies"
+  pip install ${SUPERSET_ADDITIONAL_DEPENDENCIES}
+  echo_step "1" "Complete" "Install additional dependencies"
+else
+    echo_step "1" "Skipping" "Install additional dependencies"
 fi
+
+ADMIN_PASSWORD="admin"
 # Initialize the database
-echo_step "1" "Starting" "Applying DB migrations"
```

Comment:
If the only preparation step is going to be running migrations, I think we can drop the `echo_step` stuff. We don't print those with Airflow either.

Reply:
@DouweM agreed if thats all we end up using the start up script for.